packages/node/src/exec/default-attenuator.js

```diff
@@ -69,6 +73,16 @@ export const makeGlobalsAttenuator = (
     }
   }
 
+  if (typeof scuttleGlobalThis === 'boolean') {
+    scuttleGlobalThis = {enabled: scuttleGlobalThis}
+  }
+  if (scuttleGlobalThis?.enabled) {
+    if (!isArray(scuttleGlobalThis?.exceptions)) {
+      scuttleGlobalThis.exceptions = []
+    }
+    scuttleGlobalThis.exceptions.push(...SCUTTLE_EXCEPTIONS)
```

Comment:
are duplicates in the `exceptions` array going to cause any problems?

Reply:
no because the loop attempts a scuttle only if the property is configurable, so the 2nd time it goes through it, it won't be, so it gets skipped